frontend/src/types/index.ts

```diff
@@ -61,15 +61,20 @@ export interface Module {
   guidance?: string;
   response?: boolean; // User's response
   minimumRisk?: string;
-  checkLists: {[checklistName: string]: ChecklistItem[]};
+  checkLists: Checklists;
```

Comment:
@zsims might want to check me on this, but I think you can simplify this with 

```ts
checkLists: Record<string, ChecklistItem[]>;
```

Reply:
I'd prefer to keep the Checklist Object as it makes it a little clearer and i use the object elsewhere too for QuickChecklists.